packages/connectkit/src/wallets/walletConfigs.tsx

```diff
@@ -65,7 +65,19 @@ export const walletConfigs: {
         : `https://argent.link/app/wc?uri=${encodeURIComponent(uri)}`;
     },
   },
-  'coinbaseWallet, coinbaseWalletSDK': {
+  bloom: {
+    name: 'Bloom',
+    icon: <Logos.Bloom />,
+    downloadUrls: {
+      download: 'https://connect.family.co/v0/download/bloom',
+      website: 'https://bloomwallet.io/',
+      desktop: 'https://bloomwallet.io/',
+    },
+    getWalletConnectDeeplink: (uri: string) => {
+      return `bloom://wallet-connect/wc?uri=${encodeURIComponent(uri)}`;
```

Comment:
Hey @MarkNerdi, since #392 is merged, now you need to add the `shouldDeeplinkDesktop` as `true` prop.
That will have Bloom wallet working and this PR ready.